### PR TITLE
RegisterUnknown filter with regex

### PR DESCRIPTION
**Pull request by bernamaxim**
RegisterUnknown filter with regex #5023

### CODE REVIEW

File: setup/default.xml [hunk 1]
```diff
@@ -39,6 +39,8 @@
     <entry key='commands.queueing'>true</entry>
 
     <entry key='database.ignoreUnknown'>true</entry>
+    <entry key='database.registerUnknown.regex'>\w{3,15}</entry>
```
Comment (tananaev) [inline setup/default.xml:42]: Can you please move this default into the key definition instead.

Comment (bernamaxim) [reply setup/default.xml:42]: Do you mean move key to traccar.xml?

Comment (tananaev) [reply setup/default.xml:42]: No, I mean more it into the code.

Comment (bernamaxim) [reply setup/default.xml:42]: I'm sorry, Anton I cannot catch your point. 🙏
Can you give an example to make this is clear?

**Comment by tananaev** [comment]
The CI check has failed. Make sure you run the test locally first.

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**